tools/gyp/pylib/gyp/generator/msvs.py

```diff
@@ -2169,9 +2172,12 @@ def _MapFileToMsBuildSourceType(source, rule_dependencies,
   elif ext == '.rc':
     group = 'resource'
     element = 'ResourceCompile'
-  elif ext in ['.s', '.asm']:
+  elif ext in ['.asm', '.s']:
```

Comment:
Superfluous change?

Reply:
Thanks for the review!

Ah, yes. This was due to some confusion during prototyping. I'll get a rev posted with this omitted.